Animations/src/editor/ProjectScreen.cpp

```diff
@@ -13,8 +13,8 @@ namespace MathAnim
 {
 	namespace ProjectScreen
 	{
-		static ImFont* largeFont;
-		static ImFont* defaultFont;
+		static ImFont *largeFont;
```

Comment:
I think these are just auto lint settings you may have turned on. I know I don't have any linting rules set up, but I try to keep my pointer and reference syntax left-aligned `Type* ptr` `Type& ref`. It would be great if we could undo all the linting changes that applied this rule where it swaps the symbol to be right-aligned.

Reply:
Right align is the standard everywhere, for reasons dating back to the K&R that still applies today

Reply:
I don't think there's anything in C++ that's standard everywhere 🤔 

But either way, this is just a linting rule that will be solved when I set up some CI so there won't have to be discussions about this in the future.